modules/network-observability-metrics.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-metrics_{context}"]
+= Network Observability metrics
+Metrics generated by the `flowlogs-pipeline` are configurable in the `spec.processor.metrics.includeList` of the `FlowCollector` custom resource to add or remove metrics.
```

Comment:
Maybe mention that when looking for these metrics in Prometheus (e.g. in the Console through menu Observe > Metrics), all these names are prefixed "netobserv_", such as "netobserv_namespace_flows_total"

Reply:
Okay, I adjusted the wording a bit. I also feel like its insufficient to just mention configuration in the `FlowCollector` when you can also specify these metrics in a Prometheus rule, so I have both mentioned now.